### ISSUE TITLE
Incompatible updates?

### ISSUE THREAD

**Issue by rosano** [Squirrel/Squirrel.Mac#275]
I use two apps that, upon inspection, seem to include the Squirrel framework. Both self-updated to versions that are incompatible with my version of macOS (10.14). In one case I have an old DMG lying around and can downgrade easily, in the other there are apparently no archived versions and I can no longer use it unless I upgrade (which is complex for reasons).

I'm not sure how this library works, or if that's a setting the developers chose, but from a user experience perspective it can be disorienting or even traumatic to suddenly lose access to an important app in this way, especially if it would continue to work without the upgrade. Is there a use case for allowing this to happen? I feel like 'stay on the last compatible version' is also how the Mac App Store works. Also generally it's good to ask for confirmation when something 'destructive' happens. Just some thoughts, feel free to close the issue if it's not well-placed here.

**Comment by MarshallOfSound** [comment]
TLDR: this is on apps / servers to implement

https://github.com/Squirrel/Squirrel.Mac/issues/280#issuecomment-4364755484
